src/Elm/Inspector.elm

```diff
@@ -371,10 +371,10 @@ inspectInnerExpression config expression context =
         RecordExpr expressionStringList ->
             List.foldl (\a b -> inspectExpression config (Tuple.second <| Node.value a) b) context expressionStringList
 
-        RecordUpdateExpression name updates ->
+        RecordUpdateExpression name firstUpdate updates ->
             actionLambda config.onRecordUpdate
```

Comment:
I think the signature of `onRecordUpdate` should be expanded. We have the guarantee that there is a non-empty list, let's use it there as well (`( Node String, List (Node RecordSetter) )` -> `( Node String, Node RecordSetter, List (Node RecordSetter) )`. In that case `( name, firstUpdate :: updates )` would become `( name, firstUpdate,  updates )`

Reply:
Okay, I think I've fixed it. Let me know if I've missed something.